pkg/router/template/router.go

```diff
@@ -222,18 +223,61 @@ func (r *templateRouter) EnableRateLimiter(interval int, handlerFunc ratelimiter
 	glog.V(2).Infof("Template router will coalesce reloads within %v seconds of each other", interval)
 }
 
+// When there is no default cert, cobble one together from
+// the mounted secret, if present
+func secretToPem(outName string) error {
```

Comment:
Nit: Godoc should take the form of `Function does xyz`, so this should be `secretToPem composes a PEM file at the output directory from an input private key and crt file`.

Additionally, this should take a directory in which to look for the files, or a path to the each file, instead of being hardcoded in the function.